docker/Dockerfile.ci_arm

```diff
@@ -23,6 +23,9 @@ FROM ubuntu:18.04
 RUN apt-get update --fix-missing
 RUN apt-get install -y ca-certificates gnupg2
 
+# Globally disable pip cache
+RUN pip config set global.cache-dir false
```

Comment:
is it `global.no-cache-dir` or `global.cache-dir`?

Reply:
I tried it locally and it seems to work to set the global config and preventing the cache to be created.

When using it in an individual `pip install —no-cache-dir foo` it seems to be the way to disable in one instance, but I decided not going this way, because there is always a new pip install in those scripts.